tests/acceptance/server_span_test.py

```diff
@@ -61,33 +49,33 @@ def test_unsampled_request_has_no_span(thrift_obj, default_trace_id_generator):
         'zipkin.trace_id_generator': default_trace_id_generator,
     }
 
-    TestApp(main({}, **settings)).get('/sample', status=200)
+    WebTestApp(main({}, **settings)).get('/sample', status=200)
 
     assert thrift_obj.call_count == 0
 
 
 @mock.patch('py_zipkin.logging_helper.thrift_obj_in_bytes', autospec=True)
-def test_blacklisted_route_has_no_span(thrift_obj, sampled_trace_id_generator):
+def test_blacklisted_route_has_no_span(thrift_obj, default_trace_id_generator):
```

Comment:
I think you can/should delete the sampled_trace_id_generator now. That just generated a trace ID that would always be sampled, and that's not how we do sampling decisions any more after this branch.